### PR TITLE
script for creating files with test parameters

### PR DESCRIPTION
**Pull request by kis8ya**
Это скрипт для автоматизации запуска тестов в билдах тестирования pull request'ов и собранных пакетов.

reviewers: @itIGO and @uvizhe

### CODE REVIEW

File: create_params_file.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        path = os.path.expanduser(path)
+        params["packages_dir"] = os.path.abspath(path)
+    if args.elliptics_version:
+        params["elliptics_version"] = args.elliptics_version
+    return params
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--tests-config', dest="tests_config", required=True,
+                        help="path to the file with tests parameters wich will be updated.")
```
Comment (tIGO) [inline create_params_file.py]: I think "path to store tests config" is clearer.

**Comment by tIGO** [comment]
Ok, if we consider this file as temporary solution.

File: create_params_file.py (hunk 1 above)
Comment (kis8ya) [inline create_params_file.py]: I will change this option name to "path" and will change "help"-message as well.

**Comment by uvizhe** [comment]
Ок

**Pull request merged**